platform/backend/api/src/integrationTest/kotlin/io/hamal/api/http/controller/func/FuncUpdateControllerTest.kt

```diff
@@ -128,24 +110,126 @@ internal class FuncUpdateControllerTest : FuncBaseControllerTest() {
                 )
             )
             .execute()
+
         assertThat(updateFuncResponse.statusCode, equalTo(Accepted))
         require(updateFuncResponse is HttpSuccessResponse) { "request was not successful" }
 
         val req = updateFuncResponse.result(ApiFuncUpdateSubmitted::class)
         awaitCompleted(req)
-        val funcId = req.funcId
 
         with(getFunc(funcId)) {
             assertThat(id, equalTo(funcId))
             assertThat(namespace.name, equalTo(NamespaceName("createdNamespace")))
             assertThat(name, equalTo(FuncName("createdName")))
             assertThat(inputs, equalTo(FuncInputs(MapType(mutableMapOf("hamal" to StringType("createdInputs"))))))
 
-            // assertThat(code.current.version, equalTo(CodeVersion(1))) //FIXME with core-73
+            assertThat(code.current.version, equalTo(CodeVersion(1)))
+            assertThat(code.current.value, equalTo(CodeValue("createdCode")))
+
+            assertThat(code.deployed.version, equalTo(CodeVersion(1)))
+            assertThat(code.deployed.value, equalTo(CodeValue("createdCode")))
+        }
+    }
+
+    @Test
+    fun `Does not increment code version if req code is null`() {
+        val funcId = createFuncInNamespace(
+            FuncName("createdName"),
+            CodeValue("createdCode")
+        ).funcId
+
+        val updateFuncResponse = httpTemplate.patch("/v1/funcs/{funcId}")
+            .path("funcId", funcId)
+            .body(
+                ApiFuncUpdateReq(name = FuncName("updatedName"))
```

Comment:
I think we can make this explicit to increase readability
```suggestion
                ApiFuncUpdateReq(name = FuncName("updatedName"), code = null)
```